modules/apps/headless/headless-admin-taxonomy/headless-admin-taxonomy-impl/src/main/java/com/liferay/headless/admin/taxonomy/internal/resource/v1_0/KeywordResourceImpl.java

```diff
@@ -58,11 +58,45 @@
 )
 public class KeywordResourceImpl extends BaseKeywordResourceImpl {
 
+	@Override
+	public void deleteAssetLibraryKeywordByExternalReferenceCode(
+			Long assetLibraryId, String externalReferenceCode)
+		throws Exception {
+
+		AssetTag assetTag =
+			_assetTagLocalService.getAssetTagByExternalReferenceCode(
+				externalReferenceCode, assetLibraryId);
+
+		_assetTagLocalService.deleteAssetTag(assetTag);
```

Comment:
Just a question, should we use the service instead of the local one? just to be sure the user has permissions to perform that operation. It applies also to the rest of the methods

Reply:
yes, till now was not needed but now I will implement this.

Thanks!